homeassistant/components/hue/sensor_base.py

```diff
@@ -280,5 +280,7 @@ class GenericZLLSensor(GenericHueSensor):
     def device_state_attributes(self):
         """Return the device state attributes."""
         return {
-            "battery_level": self.sensor.battery
+            "battery_level": self.sensor.battery,
+            "last_updated": self.sensor.state['lastupdated'],
+            "hue_id": self.sensor.id
```

Comment:
how is this different than `self.device_id`?

Reply:
If we want to manipulate the objects in rest api it is necessary to recover the unique identifier in the bridge Hue and not the identifier of the device

![HueScreenshot](https://user-images.githubusercontent.com/1258123/59549514-fca6d300-8f5e-11e9-995a-936462d99f09.png)

Reply:
What is the use of including it?